src/Strings.ts

```diff
@@ -8,8 +8,7 @@ import * as _ from "lodash";
 
 const unicode = require("unicode-properties");
 
-function computeAsciiMap(
-    mapper: (codePoint: number) => string
+function computeAsciiMap(mapper: (codePoint: number) => string
```

Comment:
Remove.

Reply:
What should I remove in this block?

Reply:
The change only reformats the code, unless I'm super blind.